src/state/schedule-data/schedule-data.action-creator.ts

```diff
@@ -143,53 +144,30 @@ export class ScheduleDataActionCreator {
   }
 
   static updateSchedule(newScheduleModel: ScheduleDataModel): ScheduleActionModel {
-    // TODO: make separate action creator for Tmp
     return {
       type: createActionName(TEMPORARY_SCHEDULE_NAME, ScheduleActionType.UPDATE),
       payload: newScheduleModel,
     };
   }
 
-  static addNewShift(shift: Shift): (dispatch) => Promise<void> {
-    return async (dispatch): Promise<void> => {
-      const action = {
-        type: ScheduleActionType.ADD_NEW_SHIFT,
-        payload: { ...shift },
-      };
-      dispatch(action);
-    };
+  static addNewShift(shift: Shift) {
+    throw Error("Not implemented");
```

Comment:
Nie obsługejmy tego w stanie, więc na razie `Error`.
Jak będziemy nad tym pracować to dodamy implementację